src/plugins/content/accessibility/jumplinks/jumplinks.tpl

```diff
@@ -1,15 +1,15 @@
 <nav class="jump-links-box" aria-label="{{__ "Jump Menu"}}">
     <ul>
         <li class="jump-link-item">
-            <button data-jump="question" class="jump-link" >{{__ "Jump to:"}} <b>{{__ "Question"}}</b></button>
+            <button data-jump="question" class="jump-link" >{{__ "Jump to:"}} <b>{{__ "Question"}} - {{__ questionStatus}}</b></button>
```

Comment:
Translation requires pure string. Using a variable or any other dynamic input will prevent the scanner to fetch the text. For such situation, text should be translated upfront.

Reply:
Never thought about it. Thx